app/views/courses/show.html.erb

```diff
@@ -5,127 +5,198 @@
+              course_path(@course, locale: l),
+              class: "#{'active' if l == params[:locale]} dropdown-item " %>
+        <% end %>
+      </div>
+    </div>
+  </div>
 
-      <div class="col-md-9 order-2 <%= text_direction -%>" dir="<%= text_direction -%>">
-        <h2 class="<%= language_class -%>" lang="<%= LocalesService.current %>">
-          <%= tc(@course.name) %>
-        </h2>
+  <div class="col my-5">
+    <h2 class="text-center <%= language_class -%>">
+      <%= tc(@course.name) %>
+    </h2>
+    <p class="lead mx-auto text-center <%= language_class -%>">
+      <%= @course.summary %>
+    </p>
+  </div>
 
-        <div class="lead mb-3 <%= language_class -%>" lang="<%= LocalesService.current %>">
-          <%= @course.summary %>
-        </div>
+  <div class="row bg-success">
+    <div class="col-sm-4 my-auto">
+      <%=
+        image_tag @course.image,
+          class: 'img-fluid w-100',
+          alt: @course.summary
+      %>
+    </div>
+    <div class="col-sm-8 p-5 course-description <%= text_direction -%> <%= language_class -%>" lang="<%= LocalesService.current %>">
+      <%= sanitize(tc(@course.description)) %>
+    </div>
+  </div>
 
-        <div class="mb-3">
-          <% if @subscription.present? %>
-            <div class="alert alert-info" role="alert">
-              <% if @subscription.active? %>
-                <%= t('courses.show.active_subscription') %>
-              <% else %>
-                <%= t('courses.show.paused_subscription') %>
-              <% end %>
+  <div class="row my-4 mx-auto w-75">
+      <h4 class="mx-auto text-center">Our bite-sized information is put together in
+        digestible steps called notes. Follow these below.</h4>
+      <% if @subscription.present? %>
+        <h5 class="mx-auto text-center">You already have a subscription for this pathway.
+          Make it work for you by
+          <%=
+            link_to(
+              "editing your subscription.",
+              course_subscription_path(@course),
+            )
+          %>
+        </h5>
+      <% elsif current_user? %>
+        <h5 class="mx-auto text-center">To get the notes for this pathway emailed directly to you
+          <%=
+            link_to(
+              "sign up for a subscription.",
+              course_subscription_path(@course),
+            )
+          %>
+        </h5>
+      <% else %>
+        <h5 class="mx-auto text-center">To track your progress or to get bite-sized
+          content emailed directly to you
+          <%=
+            link_to(
+              "create an account",
+              course_subscription_path(@course),
+            )
+          %>
+          or
+          <%=
+            link_to(
+              "sign in.",
+              course_subscription_path(@course),
+            )
+          %>
+        </h5>
+      <% end %>
+  </div>
 
-              <%=
-                link_to t('phrases.edit').capitalize,
```

Comment:
Can now remove the `phrases.edit` key from the translations file as I believe it's not being used anywhere else.

(I was going to mention that all strings should be pushed to the translation files, but given it's unlikely we'll ever translate the site itself anytime soon there's probably no point :))

Reply:
Oh sorry, ignore this comment – it _is_ being used in the subscription card partial.